internal/pkg/cli/app_init.go

```diff
@@ -256,6 +273,31 @@ func (o *initAppOpts) askDockerfile() error {
 	return nil
 }
 
+func (o *initAppOpts) askAppPort() error {
+	if o.AppPort != 0 {
+		return nil
+	}
+
+	port, err := o.prompt.Get(
+		fmt.Sprintf(fmtAppInitAppPortPrompt),
+		fmt.Sprintf(fmtAppInitAppPortHelpPrompt),
+		validateApplicationPort,
+		prompt.WithDefaultInput("80"),
+	)
+	if err != nil {
```

Comment:
Can we remove the "failed to" in the error logs? https://github.com/uber-go/guide/blob/master/style.md#error-wrapping